tools/amazon_agent_settings/agent_gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+gem "manageiq-gems-pending", ">0", :require => "manageiq-gems-pending", :git => "https://github.com/ManageIQ/manageiq-gems-pending.git", :branch => "master"
+gem "manageiq-smartstate", "~>0.1.5", :require => "manageiq-smartstate"
+gem "amazon_ssa_support", ">0", :require => "amazon_ssa_support", :git => "https://github.com/ManageIQ/amazon_ssa_support.git", :branch => "master"
```

Comment:
I assume we'll cut a version of this and when we do we should lock down to a specific version number.  Otherwise you could release a breaking change in the future and it will break every SSA agent out there.

Reply:
Ok, Will set to specific version for these gems.